src/main/java/app/controller/Settings.java

```diff
@@ -0,0 +1,130 @@
+package app.controller;
+
+import java.awt.geom.Rectangle2D;
+
+public class Settings
+{
+    private int gamemode;
+    private int height;
+    private int width;
+    private int noOfGuards;
+    private int noOfIntruders;
+    private double walkSpeed;
+    private double sprintSpeed;
+    private double timeStep;
+    private double scaling;
+    private Rectangle2D[] walls;
```

Comment:
Consider making these `ArrayLists` where you can keep adding them as you parse the file, if you make the getter return a copy, rather than the reference you can keep it as immutable.